src/partials/query.editor.html

```diff
@@ -10,10 +10,6 @@
         <select class="gf-form-input gf-size-auto" ng-model="ctrl.target.resultFormat" ng-options="f.value as f.text for f in ctrl.resultFormats" ng-change="ctrl.refresh()"></select>
       </div>
     </div>
-    <div class="gf-form max-width-25"  ng-hide="ctrl.target.resultFormat === 'table'">
-      <label class="gf-form-label query-keyword">ALIAS BY</label>
-      <input type="text" class="gf-form-input" ng-model="ctrl.target.alias" spellcheck='false' placeholder="Naming pattern" ng-blur="ctrl.refresh()">
-    </div>
```

Comment:
Why do you need this change?

Reply:
This has been removed because it is unimplemented, unnecessary, and confusing.

Reply:
Got it. I checked it's similar with InfluxDB plugin but not work.